### DISCUSSION TITLE
Version 1.1.5

### DISCUSSION THREAD

**Discussion by HarshPatel5940** [HarshPatel5940/Discord-Helper-Bot#31] [Announcements]
<h1 align="center"> Bot Version 1.1.4 </h1>

### New List of TODO

- [ ] Reloading Commands
- [ ] 

---

## What's Changed
* Working on v1.1.5 by @HarshPatel5940 in https://github.com/HarshPatel5940/Discord-Helper-Bot/pull/30
   -  ✅ Added Error Handling
   -  🎫 New Cmd To delete closed tickets
   -  Bug Fixing
   - Added Awaits and made things async

* Update index.md by @HarshPatel5940 in https://github.com/HarshPatel5940/Discord-Helper-Bot/pull/28
* syncing changes by @HarshPatel5940 in https://github.com/HarshPatel5940/Discord-Helper-Bot/pull/29

**Full Changelog**: https://github.com/HarshPatel5940/Discord-Helper-Bot/compare/v1.1.4...v1.1.5

<hr /><em>This discussion was created from the release <a href='https://github.com/HarshPatel5940/Discord-Helper-Bot/releases/tag/v1.1.5'>Version 1.1.5</a>.</em>

**Comment by Azizanzz0** [comment]
Bagaimana cara membuat link raw github tanpa isi linknya tidak dapat dilihat, dan saat di lihat isi rawnya 404: not found, namun rawnya dapat diakses loadstring script roblox

**Reply by HarshPatel5940** [reply]
> Bagaimana cara membuat link raw github tanpa isi linknya tidak dapat dilihat, dan saat di lihat isi rawnya 404: not found, namun rawnya dapat diakses loadstring script roblox

Why?
